test/powershell/Modules/Microsoft.PowerShell.Core/PSSessionConfiguration.Tests.ps1

```diff
@@ -488,7 +492,7 @@ namespace PowershellTestConfigNamespace
 
                     $Result.Session.Name | Should be $TestSessionConfigName
                     $Result.Session.SessionType | Should be "Default"
-                    $Result.Session.PSVersion | Should be 6.0
+                    $Result.Session.PSVersion | Should BeExactly $expectedPSVersion
```

Comment:
I am still not convinced changing this from a string literal to the variable is the right move. Please provide answers to the question in the other PR.

Reply:
The tests should work on all PowerShell Core versions without changes.
We don't pass-through the version constant, we evaluate it. So we should have a contract for this in the tests.